Microsoft.Sbom.sln

```diff
@@ -54,6 +54,9 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Sbom.Targets.Tests", "test\Microsoft.Sbom.Targets.Tests\Microsoft.Sbom.Targets.Tests.csproj", "{E31B914C-F24B-4DC8-ACC7-CAEA952563B8}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Sbom.Tool.Tests", "test\Microsoft.Sbom.Tool.Tests\Microsoft.Sbom.Tool.Tests.csproj", "{FC5A9799-7C44-4BFA-BA22-55DCAF1A1B9F}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
This section makes the E2E tests run _after_ the DotNetTool project is fully built. Without it, we have a race condition where the E2E tests and the DotNetTool project build can compete over a the NuGet package that the DotNetTool package produces. Changes were made through Visual Studio's project dependencies UI